src/python/pants/backend/jvm/tasks/jvm_compile/zinc/zinc_compile.py

```diff
@@ -37,6 +40,16 @@
 
 logger = logging.getLogger(__name__)
 
+class NailgunClientBinary(Script):
+  options_scope = 'nailgun-client'
+  name = 'ng'
+  default_version = '1.0.0'
+
+  def get_external_url_generator(self):
+    class NailgunClientBinaryUrlGenerator(BinaryToolUrlGenerator):
+      def generate_urls(self, version, host_platform):
+        return ["https://github.com/facebook/nailgun/releases/download/nailgun-all-v1.0.0/ng.py"]
```

Comment:
Would this need to be an option with this url as the default to accommodate for internal consumption?